### PR TITLE
Fix coverage reporting

### PR DESCRIPTION
**Pull request by dstansby**
I'm not sure why coverage reporting is at 0% at the moment, but hopefully this will fix it? I also renamed `build.yml` > `test.yml` to be clearer on what that file actually does.

### CODE REVIEW

**Comment by dstansby** [comment]
🎉 it finally worked!

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 [![Tests](https://github.com/quantumjot/BayesianTracker/actions/workflows/build.yml/badge.svg)](quantumjot/BayesianTracker/actions)
```
Comment (quantumjot) [inline README.md:3]: Do you need to change this since you renamed the workflow?

Comment (dstansby) [reply README.md:3]: Ooh yes, thanks for catching!

Comment (quantumjot) [reply README.md:3]: np - I was just trying to work out what wizardy was required to get this to work!

Comment (dstansby) [reply README.md]: The actual fix was putting `envsitepackagesdir` into `--cov={envsitepackagesdir}/btrack`, so `pytest-cov` is running coverage against the package installed in the tox environment, not the local checkout of the package.

**Review by paddyroddy — APPROVED** [review]

**Pull request merged**